.github/workflows/ci.yml

```diff
@@ -18,7 +18,6 @@ jobs:
           - '3.10'
           - '3.9'
           - '3.8'
-          - '3.7'
```

Comment:
There's 3 instances of `matrix.python != '3.7'` at lines 50, 63 and 66.
Could you remove those as well

Reply:
I removed the remaining references you pointed out. Thanks!